modules/rte_mis_home/rte_mis_home.module

```diff
@@ -76,6 +93,11 @@ function rte_mis_home_preprocess_page(array &$variables) {
       ];
       break;
   }
+
+  // Add a class if the current path is /about-us.
+  if ($alias == '/about-us') {
+    $variables['notification'] = TRUE;
+  }
```

Comment:
Why we are adding this in preprocess page hook?

Reply:
Removed the if block as it is already mentioned in the preprocess node.